[
  {
    "file": "/usr/local/zig/zig-macos-aarch64-0.11.0-dev.2939+289234744/lib/std/debug.zig",
    "line": 285,
    "column": 13,
    "message": "panic: reached unreachable code\nin assert (test)",
    "severity": 0
  },
  {
    "file": "/Users/ianic/code/vscode/zig_extras/test_project/src/main.zig",
    "line": 50,
    "column": 20,
    "message": "panic: reached unreachable code\nin third (test)",
    "severity": 0
  },
  {
    "file": "/Users/ianic/code/vscode/zig_extras/test_project/src/main.zig",
    "line": 45,
    "column": 9,
    "message": "panic: reached unreachable code\nin second (test)",
    "severity": 0
  },
  {
    "file": "/Users/ianic/code/vscode/zig_extras/test_project/src/main.zig",
    "line": 41,
    "column": 10,
    "message": "panic: reached unreachable code\nin first (test)",
    "severity": 0
  },
  {
    "file": "/Users/ianic/code/vscode/zig_extras/test_project/src/main.zig",
    "line": 37,
    "column": 9,
    "message": "panic: reached unreachable code\nin test.deep assert (test)",
    "severity": 0
  },
  {
    "file": "/usr/local/zig/zig-macos-aarch64-0.11.0-dev.2939+289234744/lib/test_runner.zig",
    "line": 101,
    "column": 28,
    "message": "panic: reached unreachable code\nin mainServer (test)",
    "severity": 0
  },
  {
    "file": "/usr/local/zig/zig-macos-aarch64-0.11.0-dev.2939+289234744/lib/test_runner.zig",
    "line": 35,
    "column": 25,
    "message": "panic: reached unreachable code\nin main (test)",
    "severity": 0
  },
  {
    "file": "/usr/local/zig/zig-macos-aarch64-0.11.0-dev.2939+289234744/lib/std/start.zig",
    "line": 598,
    "column": 21,
    "message": "panic: reached unreachable code\nin main (test)",
    "severity": 0
  }
]